src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,108 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int DEFAULT_SIZE = 0;
+    private T removedValue;
```

Comment:
let's make this a local variable - it is not related to the state of the arraylist